tools/github/README.md

```diff
@@ -10,83 +10,86 @@ This area is focused on the management of the Azure SDK GitHub repositories, con
 
 ## Structure
 
-* **root**  
```

Comment:
You done messed with my formatting.   I want to keep the line break, please.

Reply:
Commonmark elides those when rendered. markdown-lint was showing a lot of warnings, which tends to hide other - potentially valid - warnings.

What line break are you talking about anyway? All the space at the beginning would've been elided out.

Reply:
I think the double space at the end of a line forces a line break when rendered

Reply:
![image](https://github.com/Azure/azure-sdk-tools/assets/1291634/488c8395-4c71-45ef-96af-7d6d48e81897)

vs

![image](https://github.com/Azure/azure-sdk-tools/assets/1291634/99412b63-6098-4477-84ed-7cf5be1c59d8)

Reply:
That's the "wrong" way to do it, though. Git itself doesn't like trailing whitespace, and to get a line break in markdown the way that actually looks closer to how it renders (so it's readable in plain text form as well) is a separate line break e.g.,
```markdown
- foo

  bar
```
Whether using `-` (or `*`, but markdown-lint prefers `-` which is more generally what we use anyway, like even in this file) or `1.`, the number of whitespace characters has to match the first rendered line char e.g., 2 spaces for `-`.

Reply:
@jsquire and I clarified what he meant offline, though, so this is already fixed.

Reply:
I'm still seeing no line break.  It's rendering inline as Patrick's graphic shows.   Is there a pending change that needs to be pushed?